buildscripts/kokoro/unix.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 
 # This file is used for both Linux and MacOS builds.
+# For Linux, this script is called inside a docker container with
+# the correct environment for releases.
 # To run locally:
 #  ./buildscripts/kokoro/unix.sh
+# For 32 bit:
+#  ARCH=32 ./buildscripts/kokoro/unix.sh
 
 # This script assumes `set -e`. Removing it may lead to undefined behavior.
 set -exu -o pipefail
 
+readonly GRPC_JAVA_DIR="$(cd $(dirname $(readlink -f "$0"))/../...; pwd)"
```

Comment:
Is readlink -f available on OS X? You're removing a comment below saying it isn't.

Reply:
I checked the osx logs and this command failed. The only reason the build didn't fail was the `;`. Fixed both.